flycheck.el

```diff
@@ -6241,6 +6242,39 @@ See URL `https://github.com/kisielk/errcheck'."
     ;; packages, and can't check individual Go files.
     (and (flycheck-buffer-saved-p) (flycheck-go-package-name))))
 
+(defun flycheck-groovy-write-compiler ()
+  "Write the compilation program to temporary file so it can be executed."
+  (let* ((dir (flycheck-temp-dir-system))
+         (file (expand-file-name "compiler.groovy" dir))
+         (prog "import org.codehaus.groovy.control.*
+
+file = new File(args[0])
+unit = new CompilationUnit()
+unit.addSource(file)
+
+try {
+    unit.compile(Phases.CONVERSION)
+} catch (MultipleCompilationErrorsException e) {
+    e.errorCollector.write(new PrintWriter(System.out, true), null)
```

Comment:
Nope. ErrorCollector cannot, but since I'm writing the program that groovy will execute, I can dump JSON if I wanted to. I just wrote the first thing that worked and had the most examples in flycheck.el source code for reference.

Why do you ask?